### ISSUE TITLE
Questions about checkpoints and loss functions

### ISSUE THREAD

**Issue by mya33** [SAWRJJ/UWSegFormer#2]
I have two questions.
The first is about the checkpoints in Google Drive.
I used the checkpoints in Google Drive and test.py to check the mIoU for the SUIM and DUT datasets.
I confirmed that the mIoU for the DUT dataset matches that of the paper.
(At this time, I changed the number of heads from 2 to 4.)
On the other hand, the mIoU for the SUIM dataset is lower (80.6%) compared to the paper.
Please let me know if there are any modifications needed to achieve the same accuracy as the paper for the SUIM dataset.

The second is about the loss function.
The paper uses cross-entropy and ELL, but the source code seems to use only cross-entropy.
Is the checkpoint in Google Drive trained using only cross-entropy?
Also, the ELL implementation is actually ABL, but would using ABL improve accuracy?
